wiki/Medals/Legacy_Beatmap_Packs/id.md

```diff
@@ -0,0 +1,406 @@
+# Beatmap pack lama
+
+Beatmap pack telah menjadi bagian yang tak terpisahkan dari komunitas osu! sejak [pengenalan pertamanya](https://osu.ppy.sh/community/forums/topics/1853) oleh ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) pada 2008, di mana beberapa pack terikat pada beberapa [achievement](/wiki/Medals). Namun dengan perkembangan osu! dan komunitasnya yang terus-menerus setiap tahun, mapping juga telah mengalami perubahan yang sangat besar dan—karena permintaan yang besar dari publik—[pada 16 Januari 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) perubahan besar dilakukan pada beberapa beatmap pack untuk membuat beatmap pack tetap relevan waktu.
```

Comment:
`Paket beatmap telah menjadi bagian yang tak terpisahkan dari komunitas osu! sejak [pertama kali diperkenalkan]() oleh [Saturos]() pada 2008, di mana pada kala itu paket-paket beatmap tertentu telah dijadikan syarat untuk dapat membuka [medali-medali yang ada](/wiki/Medals). Meskipun demikian, dengan perkembangan osu! yang semakin pesat seiring tahunnya, beatmap-beatmap yang ada juga ikut berevolusi dan — oleh karenanya — [pada 16 Januari 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) tim osu! merombak ulang isi berbagai paket beatmap yang ada agar tidak lekang termakan zaman.`
`

Reply:
`—karena permintaan yang besar dari publik—` is additional adverb so we use [tanda pisah](https://puebi.js.org/tanda/pisah) to include it in the sentence. Here is the revision:
`Paket beatmap telah menjadi bagian yang tak terpisahkan dari komunitas osu! sejak [pertama kali diperkenalkan](https://osu.ppy.sh/community/forums/topics/1853) oleh ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) pada 2008, di mana pada kala itu paket-paket beatmap tertentu telah dijadikan syarat untuk dapat membuka [medali-medali yang ada](/wiki/Medals). Meskipun demikian, dengan perkembangan osu! yang semakin pesat seiring tahunnya, beatmap-beatmap yang ada juga ikut berevolusi dan—karena permintaan yang besar dari publik—[pada 16 Januari 2016](https://osu.ppy.sh/home/news/2016-01-18-osuweekly-44) tim osu! merombak ulang isi berbagai paket beatmap yang ada agar tidak lekang termakan zaman.`